cloudnet-modules/cloudnet-cloudflare/src/main/java/de/dytanic/cloudnet/ext/cloudflare/CloudflareAPI.java

```diff
@@ -61,15 +61,15 @@ public static CloudflareAPI getInstance() {
             HttpURLConnection httpURLConnection = (HttpURLConnection) new URL(CLOUDFLARE_API_V1 + "zones/" + zoneId + "/dns_records").openConnection();
             httpURLConnection.setRequestMethod("POST");
 
-            return this.getJsonResponse(httpURLConnection, email, apiKey, dnsRecord, serviceName, zoneId);
+            return this.getJsonResponse(httpURLConnection, email, authenticationMethod, apiKey, dnsRecord, serviceName, zoneId);
         } catch (Exception exception) {
             exception.printStackTrace();
         }
 
         return null;
     }
 
-    public Pair<Integer, JsonDocument> updateRecord(String serviceName, String email, String apiKey, String zoneId, String recordId, DNSRecord dnsRecord) {
+    public Pair<Integer, JsonDocument> updateRecord(String serviceName, String email, CloudflareConfigurationEntry.AuthenticationMethod authenticationMethod, String apiKey, String zoneId, String recordId, DNSRecord dnsRecord) {
         Preconditions.checkNotNull(email);
```

Comment:
Same as above

> These check should only be done when using the AuthenticationMethod Global-Key, because the email is not needed when using the Bearer Token (could be set to null, although an empty string is working fine)